lib/rules/indentation/index.js

```diff
@@ -628,7 +630,7 @@ function inferRootIndentLevel(root, baseIndentLevel, indentSize) {
 	}
 
 	const indents = [];
-	const foundIndents = /(?:^|\n)([ \t]*)\S/m.exec(root.raws.beforeStart);
+	const foundIndents = root.raws.codeBefore && /(?:^|\n)([ \t]*)\S/m.exec(root.raws.beforeStart);
```

Comment:
[suggestion] Isn't it needed to use `beforeStart`?

```suggestion
	const foundIndents = root.raws.codeBefore && /(?:^|\n)([ \t]*)\S/m.exec(root.raws.codeBefore);
```

Reply:
ah good catch! just missed it